sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -96,13 +96,21 @@
 
 function samePixel(array, index, refPixel, row, pixelTag)
 {
-    if (array[index] == refPixel[0] &&
-        array[index + 1] == refPixel[1] &&
-        array[index + 2] == refPixel[2] &&
-        array[index + 3] == refPixel[3]) {
+    if (array[index] == refPixel[0][0] &&
+        array[index + 1] == refPixel[0][1] &&
+        array[index + 2] == refPixel[0][2] &&
+        array[index + 3] == refPixel[0][3]) {
         return true;
     }
-    testFailed(pixelTag + " pixel of row " + row + ": expected " + refPixel + ", got " +
+    if (refPixel.length == 2 &&
+        array[index] == refPixel[1][0] &&
+        array[index + 1] == refPixel[1][1] &&
+        array[index + 2] == refPixel[1][2] &&
+        array[index + 3] == refPixel[1][3]) {
+        return true;
+    }
```

Comment:
This code could easily be generalized to support arbitrarily many entries in refPixel by iterating from 0 to refPixel.length - 1. I think you should do that.